quickjs/src/jniMain/kotlin/app/cash/quickjs/ziplineJni.kt

```diff
@@ -54,9 +52,11 @@ actual abstract class Zipline {
 
   companion object {
     fun create(
-      dispatcher: CoroutineDispatcher = Executors.newSingleThreadExecutor().asCoroutineDispatcher(),
+      dispatcher: CoroutineDispatcher,
```

Comment:
And just like that you've made this whole file compatible with Kotlin/Native. Saved me an expect/actual function to produce a default argument value.

Reply:
Nice.